CONTRIBUTING.md

```diff
@@ -40,3 +40,32 @@ Before you submit your pull request, please review the following:
 
 I will try to help you get the PR in mergeable shape within a reasonable time, but it may take a few days.
 It is best if you check your GitHub notifications in the meantime!
+
+## Building docs locally
+
+Documentation is built using:
+
+- <https://www.mkdocs.org/#installing-mkdocs>
+- <https://squidfunk.github.io/mkdocs-material/getting-started/#installation>
+- <https://github.com/mkdocs/mkdocs-redirects>
+
+Quick build via:
+
+```bash
+python3 -m pip install --upgrade pip setuptools build
+python3 -m pip install -r ./docs/src/requirements.txt
+```
+
+Docs layout:
```

Comment:
Missing info about how to start local dev server. It was included in the old readme.